### ISSUE TITLE
-packet-streamer-endpoint not recognized by emulator

### ISSUE THREAD

**Issue by mickael9** [google/bumble#208]
`-packet-streamer-endpoint` does not seem to be supported with latest Android emulator (32.1.13)

If there is an alternative, can this me mentioned in the docs?

Thanks

**Comment by mickael9** [comment]
Nevermind, this requires bleeding edge emulator from Canary update channel:

```
$ tools/bin/sdkmanager --channel=3 emulator
```

**Comment by vChavezB** [comment]
Hi @mickael9 , I am trying to test the android emulator with netsim. What are the steps I need to verify that it is working?

I installed the latest emulator (v33.1.1) through the Canary release of android studio. Then I created a device with SDK API 33. When I try to turn on bluetooth on the emulator the bluetooth of the emulator is turned off automatically and then tries to turn on back again and again ....

`emulator -avd Wear_OS_Large_Round_API_33 -packet-streamer-endpoint default -grpc 1234`
